modules/s3db/req.py

```diff
@@ -2122,13 +2122,15 @@ def model(self):
 
         list_fields = ["id",
                        "skill_id",
-                       # @ToDo: Activate based on a deployment_setting
-                       #"task",
                        "quantity",
                        "quantity_transit",
                        "quantity_fulfil",
                        "comments",
                        ]
+
+        if settings.get_req_task_list_field():
+            list_fields.insert(2, "task")
```

Comment:
Ok, from the comment up in the req_req_skill table definition...
#Field("task",
# writable=False, # Populated from req_req 'Purpose'
this is nothing to do with project_task.  It means the "purpose" field in the req_req table.  The req_req_skill table already has a foreign key ref to the req_req table, so no need for a copy of that field here with a misleading name...  We took the confusing comment out, as the field is not needed.  Caching the "purpose" text here, to avoid a lookup (?) is not sufficient to justify the added complication.